src/Sylius/Bundle/ResourceBundle/Controller/ResourceController.php

```diff
@@ -166,6 +168,8 @@ public function indexAction(Request $request)
             ->setData($resources)
         ;
 
+        $view->getSerializationContext()->enableMaxDepthChecks();
```

Comment:
This should be done in `handleView()` same as there is group setting being made.